easybuild/tools/py2vs3/py3.py

```diff
@@ -62,6 +62,12 @@
 # string type that can be used in 'isinstance' calls
 string_type = str
 
+warning_lines = [
+    "Using py2vs3 is deprecated, since EasyBuild no longer runs on Python 2.",
+    "This module will be removed in a future EasyBuild Version.",
+]
+sys.stderr.write('\n\n' + '\n'.join(warning_lines) + '\n\n\n')
```

Comment:
Let's use our `log.deprecated` for this: https://github.com/branfosj/easybuild-framework/pull/7